lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -142,7 +142,7 @@ const resolveTokenOnlyIfOutOfIframe = (target: any, propertyKey: string, descrip
 export class UserAgentApplication {
 
   // input Configuration by the user
-  private config: Configuration;
+  public config: Configuration;
```

Comment:
why is this public?

Reply:
This is user i/p which we need for some Unit testing since many variables are unpacked in later APIs in UserAgentApplication. I think that is the reasoning for this particular change.

Reply:
Actually the dependency on this was taken out of the tests, I think it is feasible to make this private.

Reply:
I have made this private in my latest commit